docs/_docs/storage/rocksdb.adoc

```diff
@@ -0,0 +1,52 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+= RocksDB Storage
+
+RocksDB is a persistent storage engine based on LSM tree. It is best used in environments with a large number of write requests.
+
+When it is enabled for the link:config/data-region[data region], Ignite stores all the data on disk, and loads as much data as it can into RAM for processing.
```

Comment:
Same here, but I'd suggest not writing too much about regions in rocksdb, we'll remove them after the beta release